src/backend/auth/admin_access_endpoints.py

```
@@ -0,0 +1,137 @@
+            "LEFT JOIN organizations o ON o.id = i.org_id "
+            "WHERE u.status = 'pending' ORDER BY u.created_at",
+        ).fetchall()
+    return [
+        {"id": str(r["id"]), "email": r["email"],
+         "display_name": r["display_name"], "invited_org": r["invited_org"]}
+        for r in rows
+    ]
+
+
+@admin_access_router.get("/users")
+def list_users(admin: dict = Depends(require_admin)):
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)